dashboards-observability/public/components/explorer/home_table/history_table.tsx

```diff
@@ -40,44 +40,100 @@ export function Table(options: TableData) {
   };
 
   const columns = [
+    {
+      field: 'type',
+      name: '',
```

Comment:
We should add name as `type` here so that it's visible on the table.

Reply:
Ani wanted the icon column name to be empty and there's a separate column named type too for the type of the query